### PR TITLE
[hi] Localize hi/docs/partners/_index.html

### LINKED ISSUE #39872: [hi] Localize hi/docs/partners/_index.html
**This is a Feature Request**
Would like to add the Partners page directory to [hi] localization 

**What would you like to be added**
If we observe there are some of the directories that are not present in [hi] Localisation if we compare to [en] docs so to merge that gap I would like to add `content/en/partners`

### PR DESCRIPTION
**Pull request by YashPimple**
closed #39872

- [X] Localize hi/docs/partners/_index.html added the Partners page directory to [hi] localization

### CODE REVIEW

**Review by kundan2707 — COMMENTED** [review]
LGTM

File: content/hi/partners/_index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: पार्टनर्स
+bigheader: कुबेरनेट्स पार्टनर्स
+abstract: कुबेरनेट्स पारिस्थितिकी तंत्र का विकास
+class: ग्रिडपेज
+cid: पार्टनर्स
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:6]:
```suggestion
class: gridPage
cid: partners
```

File: content/hi/partners/_index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+<section id="users">
+    <h5>कुबेरनेट्स भागीदारों के साथ एक मजबूत, जीवंत कोडबेस बनाने के कार्य में इस्तेमाल किया जाता है जो पूरक प्लेटफार्मों के एक स्पेक्ट्रम का समर्थन करता है।</h5>
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:10]:
```suggestion
    <h5>कुबेरनेट्स भागीदारों के साथ एक मज़बूत, जीवंत कोडबेस बनाने के कार्य में इस्तेमाल किया जाता है जो पूरक प्लेटफार्मों के एक स्पेक्ट्रम का समर्थन करता है।</h5>
```

File: content/hi/partners/_index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      <div class="col-nav">
+        <center>
+          <h5>
+            <b>कुबेरनेट्स प्रमाणित सेवा प्रदाता</b>
+          </h5>
+          <br>गहन अनुभव वाले प्रमाणित सेवा प्रदाता उद्यमों को कुबेरनेट्स को सफलतापूर्वक अपनाने में मदद करते हैं।
+          <br><br><br>
+          <button class="button landscape-trigger landscape-default" data-landscape-types="kubernetes-certified-service-provider" id="kcsp">See KCSP Partners</button>
+          <br><br>बनने के इच्छुक हैं
+          <a href="https://www.cncf.io/certification/kcsp/">KCSP</a>?
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:21]:
```suggestion
          <button class="button landscape-trigger landscape-default" data-landscape-types="kubernetes-certified-service-provider" id="kcsp">KCSP पार्टनर्स देखें</button>
          <br><br>
          <a href="https://www.cncf.io/certification/kcsp/">KCSP</a>बनने को इच्छुक हैं?
```

File: content/hi/partners/_index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        </center>
+      </div>
+      <div class="col-nav">
+        <center>
+          <h5>
+            <b>प्रमाणित कुबेरनेट्स वितरण, होस्ट किए गए प्लेटफ़ॉर्म, और इंस्टॉलर</b>
+          </h5>सॉफ्टवेयर अनुरूपता यह सुनिश्चित करती है कि कुबेरनेट्स के प्रत्येक विक्रेता का संस्करण आवश्यक एपीआई का समर्थन करता है|
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:28]:
```suggestion
          </h5>सॉफ्टवेयर कन्फॉर्मन्स यह सुनिश्चित करती है कि कुबेरनेट्स के प्रत्येक विक्रेता का संस्करण आवश्यक एपीआई का समर्थन करता है।
```

File: content/hi/partners/_index.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          <br><br><br>
+          <button class="button landscape-trigger" data-landscape-types="certified-kubernetes-distribution,certified-kubernetes-hosted,certified-kubernetes-installer" id="conformance">See Conformance Partners</button>
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:30]:
```suggestion
          <button class="button landscape-trigger" data-landscape-types="certified-kubernetes-distribution,certified-kubernetes-hosted,certified-kubernetes-installer" id="conformance">कन्फॉर्मन्स पार्टनर्स देखें</button>
```

File: content/hi/partners/_index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+          <br><br>बनने के इच्छुक हैं
+          <a href="https://www.cncf.io/certification/software-conformance/">Kubernetes Certified</a>?
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:32]:
```suggestion
          <br><br>
          <a href="https://www.cncf.io/certification/software-conformance/">कुबेरनेट्स प्रमाणित</a>होने को इच्छुक हैं?
```

File: content/hi/partners/_index.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        </center>
+      </div>
+      <div class="col-nav">
+        <center>
+          <h5>
+            <b>कुबेरनेट्स ट्रेनिंग पार्टनर्स</b>
+          </h5>
+          <br>परीक्षण किए गए प्रशिक्षण प्रदाता जिनके पास क्लाउड नेटिव प्रौद्योगिकी प्रशिक्षण का गहरा अनुभव है|
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:40]:
```suggestion
          <br>परीक्षण किए गए प्रशिक्षण प्रदाता जिनके पास क्लाउड नेटिव प्रौद्योगिकी प्रशिक्षण का गहरा अनुभव है। 
```

File: content/hi/partners/_index.html [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+          <br><br><br>
+          <button class="button landscape-trigger" data-landscape-types="kubernetes-training-partner" id="ktp">See KTP Partners</button>
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:42]:
```suggestion
          <button class="button landscape-trigger" data-landscape-types="kubernetes-training-partner" id="ktp">KTP पार्टनर्स देखें</button>
```

File: content/hi/partners/_index.html [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+          <br><br>बनने के इच्छुक हैं
+          <a href="https://www.cncf.io/certification/training/">KTP</a>?
```
Comment (anubha-v-ardhan) [inline content/hi/partners/_index.html:44]:
```suggestion
          <br><br>
          <a href="https://www.cncf.io/certification/training/">KTP</a>बनने को इच्छुक हैं?
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by Babapool** [comment]
@YashPimple, a friendly reminder from my end. I just wanted to confirm that since there have been no updates on this PR are you still working on this PR? Thanks

**Comment by divya-mohan0209** [comment]
@YashPimple Thank you for your efforts on this! However this PR has been inactive for more than 4 months without any updates from your end. Therefore, we shall be closing it. However, if you wish to resume work on the PR whenever you have the bandwidth, please consider reopening it by issuing a /reopen command in the comments.
/close

**Pull request closed without merging**